server/ServerApp.cpp

```diff
@@ -318,7 +318,6 @@ int ServerApp::GetNewDesignID()
 
 void ServerApp::Run() {
     DebugLogger() << "FreeOrion server waiting for network events";
-    std::cout << "FreeOrion server waiting for network events" << std::endl;
```

Comment:
I didn't think of running the server process standalone.  I've restored the line.